xrpl/models/requests/request.py

```diff
@@ -61,6 +62,7 @@ class RequestMethod(str, Enum):
     NFT_INFO = "nft_info"  # clio only
     NFT_HISTORY = "nft_history"  # clio only
     NFTS_BY_ISSUER = "nfts_by_issuer"  # clio only
+
```

Comment:
extra empty line can be removed

Reply:
The line separations are on purpose, so there's spaces between the sections.

Reply:
ok